suite-common/wallet-config/src/utils.ts

```diff
@@ -0,0 +1,73 @@
+import {
+    AccountType,
+    Networks,
+    Network,
+    NetworkFeature,
+    networks,
+    networksCompatibility,
+    NetworkSymbol,
+} from './networksConfig';
+
+const networksCollection: Network[] = Object.values(networks);
+
```

Comment:
in some cases we can use `getMainnets` right away - there are some differences between `networks` and `networksCompatibility`, but a lot of it is common.
When the usecase does not rely on those properties which are different, we may use `getMainnets` right away. Those usecases which rely on network.accountType - the `getMainnetsCompatible` and `getTestnetsCompatible` allow to do it later, and have the PRs smaller.